### PR TITLE
ADJUST1-428

### PR DESCRIPTION
**Pull request by carloveo-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  val remand: RemandDto?,
+  @Schema(description = "The details of an additional days awarded adjustments (ADA)")
+  val additionalDaysAwarded: AdditionalDaysAwardedDto?,
+  @Schema(description = "Additional details of a UAL adjustment")
+  val unlawfullyAtLarge: UnlawfullyAtLargeDto?,
+  @Schema(description = "The details of a tagged-bail adjustment")
+  val taggedBail: TaggedBailDto?,
+
+  @Schema(description = "The prison where the prisoner was located at the time the adjustment was created (a 3 character code identifying the prison)", example = "LDS")
+  val prisonId: String? = null,
```
Comment (ldlharper) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt:34]: I don't think the prisonId can actually be set via the API, so I'd say it wasn't edittable.

**Review by ldlharper — COMMENTED** [review]
Can we mark the `days` and `daysBetween` within the AdjustmentDTO as deprecated. They shouldn't be needed anymore (Once the UI has been update to use the new field)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt (hunk 1 above)
Comment (carloveo-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt:34]: Its being set in the create (https://github.com/ministryofjustice/hmpps-adjustments-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsService.kt#L76) and update (https://github.com/ministryofjustice/hmpps-adjustments-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsService.kt#L195) methods

Comment (ldlharper) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt:34]: Oh yea, no idea how I missed that. Maybe it shouldn't be nullable then!

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**